.npmignore

```diff
@@ -30,3 +30,15 @@ content/files/
 
 # Since we ship the built stuff, we actually no longer need any of the source
 client/src/
+
+# Build/Dev files
+.eslint*
+.prettier*
+.stylelint*
+.flake8
+tsconfig.json
+Procfile.*
```

Comment:
Not sure if you use the `Procfile.start` as a production thing, but I guess at least the `Procfile.dev`

Reply:
Not a production thing. It's only used for local development within `yari`. 
When you use Yari from `mdn/content` (i.e. running `yarn start`) it doesn't use any of these.